src/Umbraco.Web/Models/ContentEditing/ContentTypeDisplay.cs

```diff
@@ -1,31 +1,28 @@
-﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel;
-using System.Linq;
-using System.Runtime.Serialization;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace Umbraco.Web.Models.ContentEditing
-{
-   
-    [DataContract(Name = "contentType", Namespace = "")]
-    public class ContentTypeDisplay : ContentTypeCompositionDisplay
+﻿using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Umbraco.Web.Models.ContentEditing
+{
+   
+    [DataContract(Name = "contentType", Namespace = "")]
+    public class ContentTypeDisplay : ContentTypeCompositionDisplay
     {
-        public ContentTypeDisplay()
-        {
-            //initialize collections so at least their never null
-            AllowedTemplates = new List<EntityBasic>();            
-        }
-
-        //name, alias, icon, thumb, desc, inherited from the content type
-
-        // Templates
-        [DataMember(Name = "allowedTemplates")]
-        public IEnumerable<EntityBasic> AllowedTemplates { get; set; } 
-
-        [DataMember(Name = "defaultTemplate")]
-        public EntityBasic DefaultTemplate { get; set; }
-
-    }
-}
+        public ContentTypeDisplay()
+        {
+            //initialize collections so at least their never null
+            AllowedTemplates = new List<EntityBasic>();            
+        }
+
+        //name, alias, icon, thumb, desc, inherited from the content type
+
+        // Templates
+        [DataMember(Name = "allowedTemplates")]
+        public IEnumerable<EntityBasic> AllowedTemplates { get; set; } 
+
+        [DataMember(Name = "defaultTemplate")]
+        public EntityBasic DefaultTemplate { get; set; }
+
+        [DataMember(Name = "firstContentType")]
+        public bool FirstContentType { get; set; }

```

Comment:
I don't really think this is necessary, the compositions panel already looks up all Content Types - which is essentially what this is doing. This type of data is also not really part of the 'Model'. I'll refactor and have you review.